types.go

```diff
@@ -21,6 +22,9 @@ const (
 )
 
 func (e EvaluationType) String() string {
+	if e < 0 || int(e) >= len(evaluationStrings) {
+		return fmt.Sprintf("EvaluationType(%d)", e)
```

Comment:
Only verbs 'v', 's', 'x', 'X', 'q' will call the String(). 'd' will not. Conversion is not necessary.